src/haxelib/client/Main.hx

```diff
@@ -1037,17 +1048,17 @@ class Main {
 		}
 		else
 		{
-			var inf = try site.infos(p) catch( e : Dynamic ) { Sys.println(e); return; };
-			p = inf.name;
```

Comment:
```
var latest = try site.getLatestVersion(p) catch( e : Dynamic ) { Sys.println(e); return; };
p = p.toLowerCase();
p = FileSystem.readDirectory(pdir).filter(function (dir) return dir.toLowerCase() == p)[0];
```
that would be too ugly I guess?

If only the folder was made in lower case to begin with :/